code/unclassified/src/biggest_of_n_numbers/biggest_of_n.js

```diff
@@ -0,0 +1,8 @@
+// Part of cosmos from opengenus foundation
+
+const biggestOfN = array => {
```

Comment:
Why not just use an arrow statement without braces?

```Javascript
const biggestOfN = array => Math.max.apply(null, array);
```
The return is implicit in this.